test/unit/typography/p5.Font.js

```diff
@@ -0,0 +1,28 @@
+
+suite('p5.Font', function() {
+
+  var v, myp5 = new p5(function(p) {
+    p.setup = function() {};
+    p.draw = function() {};
+  });
```

Comment:
The `var v` gets lost in this, visually—I might break it out:
```js
var myp5 = new p5(function(p) {});
var v;
```
However, I'd also vote for something like `var font` instead of `var v`